sys/include/evtimer.h

```diff
@@ -98,6 +98,20 @@ void evtimer_add(evtimer_t *evtimer, evtimer_event_t *event);
  */
 void evtimer_del(evtimer_t *evtimer, evtimer_event_t *event);
 
+/**
+ * @brief   Retrieves the time remaining for an event
+ * @warning Retrieval of evtimer event offsets disables interrupts and has
+ *          O(n) complexity with (n) being the number of active
+ *          @ref evtimer_event_t for an @ref evtimer_t
+ *
+ * @param[in] evtimer       The event timer the event belongs to
+ * @param[in] event         An event
+ *
+ * @return  The offset of the event
+ * @return  Zero when the event is not in the list
```

Comment:
Hm, I don't think this is true. In the current implementation, the offsets get added up until the event is found. If it is not found, all offsets are accumulated. right?

Reply:
You're right :(

Do you have a preference for this behaviour? I could add a simple flag to check if a result is found and change the implementation to the spec. Other option is to change the documentation to "Undefined result when the event is not in the list."

Reply:
> I could add a simple flag to check if a result is found and change the implementation to the spec. 

Probably the only way.

> Other option is to change the documentation to "Undefined result when the event is not in the list."

I think this won't work. Imagine the timer triggered after calling ```event_getoffset()```, but before that function disables the IRQs. So *when calling the function*, the timer was in the list. Still the output would be wrong. Thus the result of ```event_getoffset()``` could never be trusted...

Reply:
> I think this won't work. Imagine the timer triggered after calling event_getoffset(), but before that function disables the IRQs. So when calling the function, the timer was in the list. Still the output would be wrong. Thus the result of event_getoffset() could never be trusted...

Ack :(